### PR TITLE
update directories for use on Pachon in nublado

### PR DESCRIPTION
**Pull request by patrickingraham**
Doing this now as any changes from here on out will be major.

### CODE REVIEW

File: python/lsst/observing/constants.py [hunk 1]
```diff
@@ -6,6 +6,7 @@
 boreSight = PointD(2036.5, 2000.5)    # boreSight on detector (pixels)
 sweetSpots = {                  # the sweet spots for the gratings (pixels)
     'ronchi90lpmm' : PointD(1780, 1800),
+    'empty_1' : PointD(1780, 1800)
```
Comment (RobertLuptonTheGood) [inline python/lsst/observing/constants.py:9]: looks good

File: notebooks/new-calibrationStarVisit.ipynb [hunk 2]
```diff
@@ -18,6 +18,19 @@
     "import lsst.observing.commands.calibrationStarVisit as calibrationStarVisit"
    ]
   },
+  {
```
Comment (RobertLuptonTheGood) [inline notebooks/new-calibrationStarVisit.ipynb:21]: Can you clear all the outputs, please?

File: notebooks/new-calibrationStarVisit.ipynb [hunk 3]
```diff
@@ -67,7 +80,7 @@
     "    afwDisplay.setDefaultBackend(\"matplotlib\" if False else \"firefly\")\n",
     "    os.environ['FIREFLY_URL'] = \"http://firefly01.cp.lsst.org:8080/firefly/\"\n",
     "\n",
-    "    disp = afwDisplay.Display(1, name='RHL', reopenPlot=True)\n",
```
Comment (RobertLuptonTheGood) [inline notebooks/new-calibrationStarVisit.ipynb:70]: Do we need to commit these changes?

**Review by RobertLuptonTheGood — COMMENTED** [review]
Do we need to commit the notebook?

File: notebooks/new-calibrationStarVisit.ipynb (hunk 3 above)
Comment (patrickingraham) [reply notebooks/new-calibrationStarVisit.ipynb:70]: I'll put something generic as the name (e.g. calibrationStarVisit). Same in the snippet above regarding the network setup.

File: notebooks/new-calibrationStarVisit.ipynb (hunk 2 above)
Comment (patrickingraham) [reply notebooks/new-calibrationStarVisit.ipynb:21]: I'm a little confused here as I thought the outputs were all clear... I'll re-clear them

**Comment by patrickingraham** [comment]
The convenience of committing the notebook is due to the repo directories on the summit...
changes committed in 42a45de

**Comment by tribeiro** [comment]
can we merge this PR?

**Pull request closed without merging**